src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/claimsbyhearingvenue/ClaimsByHearingVenueReportTest.java

```diff
@@ -310,13 +298,12 @@ void shouldShowReportDetailEntriesSortedByEthosCaseReferenceAscending() {
         var expectedNumberOfSubmitEventEntries = submitEvents.size();
         var expectedFirstCaseReference = submitEvents.get(1).getCaseData().getEthosCaseReference();
```

Comment:
maybe worth changing all the vars in this file?

Reply:
removed from this file